### PR TITLE
Feature/227 change user management urls

### LINKED ISSUE #227: User and user management URLs should point to user management
Right now they're pointing to the Solution Center URL but that's wrong in the new context, where User Management is an independent application.

The best solution would be to add it to the communicator, since we can't rely on getting it from the applications returned by the backend. A user without User Management permissions won't have that application and we still need to get the URL from somewhere to go to his user account.

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #227

### CODE REVIEW

File: src/app/common/backend-connector/backend-connector.service.js [hunk 1]
```diff
@@ -24,8 +24,9 @@ class BackendConnectorService {
         && this.ScEnvironments.getCurrentEnvironment().NAME !== 'PRODUCTION';
   }
 
-  getSolutionCenterUrl() {
-    return this.environment.URL;
+  getUserManagementUrl() {
+    console.log(this.environment);
```
Comment (frontendherodk) [inline src/app/common/backend-connector/backend-connector.service.js]: Think you forgot this one :)

Comment (dami-gg) [inline src/app/common/backend-connector/backend-connector.service.js]: removed

**Comment by frontendherodk** [comment]
I've made some changes regarding the clicks. It doesn't really make sense when we can just use hrefs instead. It's more semantic.

File: src/app/components/user/user-menu/user-menu.html [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <li class="user-menu__option"
         data-e2e="scn-usermenu-item-terms">
       <a class="user-menu__option__link"
-         ng-click="$ctrl.navigateTo($ctrl.userManagementUrl + '/terms')"
+         ng-href="{{$ctrl.userManagementUrl}}/terms"
```
Comment (dami-gg) [inline src/app/components/user/user-menu/user-menu.html]: Can we use the one time binding here and in the following examples? @frontendherodk

**Comment by dami-gg** [comment]
:+1:

File: src/app/components/user/user-menu/user-menu.html (hunk 2 above)
Comment (frontendherodk) [inline src/app/components/user/user-menu/user-menu.html]: Ah yes. I'll quickly fix that

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**